index.js

```diff
@@ -135,7 +144,7 @@ function launchSpec(kernelSpec, spawnOptions) {
  *                                        package `kernelspecs`
  * @param  {object}       config          connection config
  * @param  {string}       connectionFile  path to the config file
- * @param  {object}       [spawnOptions]  options for [child_process.spawn]{@link https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options}
+ * @param  {object}       [spawnOptions]  `child_process`-like options for [execa]{@link https://github.com/sindresorhus/execa#options}
```

Comment:
How much does it overlap? Would we call this a major version change? Do we need to adapt code bases for nteract and hydrogen?

Reply:
> Would we call this a major version change? Do we need to adapt code bases for nteract and hydrogen?

It should be a minor bump only. `execa` seems to have been designed for backward compatibility with `child_process`.